src/planwise/model/import_job.clj

```diff
@@ -390,6 +390,10 @@
                          :type-field type-field)]
     (import-job job-value)))
 
+(defn restore-job
+  [job]
+  (import-job (:state job) (:value job)))
```

Comment:
Yep, it fails when having multiple facilities processing tasks running at the same time, since `dispatch-process-facilities` is the only dispatch action that actually modifies the job. I'll have to review this then.